OutOfSchool/OutOfSchool.DataAccess/Models/EmailOutbox.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace OutOfSchool.Services.Models;
+public class EmailOutbox : IKeyedEntity<Guid>
+{
+    public Guid Id { get; set; }
+
+    public string Email { get; set; }
+
+    public string Subject { get; set; }
+
+    public string HtmlContent { get; set; }
+
+    public string PlainContent { get; set; }
+
+    public DateTime CreationTime { get; set; }
```

Comment:
it depends on implementation of the quartz job (will it delete an email after sending or not), but for consideration: can add some metadata fields for debugging/analysis, like the sent (delivered) time, expiration time (for example if we send an email with some token we probably should not send it if it was postponed till the token has expired), optionally store error why it was not sent (we can get it in logs so don't know if should store in db)